test/asset/ProjectWithPsalmBaseline/infection.html

```diff
@@ -0,0 +1,19 @@
deCoverage\nError: Call to protected method Acme\\ProjectWithPsalmBaseline\\HelloWorld::takesImplictMixedReturnsIntAndIsCoveredByBaseline() from scope HelloWorldTest\n\n\/home\/barney\/projects\/infection-static-analysis-plugin\/test\/asset\/ProjectWithPsalmBaseline\/tests\/HelloWorldTest.php:13\n\nERRORS!\nTests: 1, Assertions: 0, Errors: 1.","coveredBy":["97ea2e5ee1e0a71c3f2a240c57ba2162"],"killedBy":["97ea2e5ee1e0a71c3f2a240c57ba2162"],"testsCompleted":1},{"id":"362c892a8ec7d8782c4427adbed5607c","mutatorName":"DecrementInteger","replacement":"return $mixture + 1;","description":"Decrements an integer value with 1.","location":{"start":{"line":12,"column":9},"end":{"line":12,"column":29}},"status":"Killed","statusReason":"PHPUnit 10.5.1 by Sebastian Bergmann and contributors.\n\nRuntime:       PHP 8.1.12-1ubuntu4.3\nConfiguration: \/tmp\/infection\/phpunitConfiguration.362c892a8ec7d8782c4427adbed5607c.infection.xml\n\n.                                                                   1 \/ 1 (100%)\n\nTime: 00:00.002, Memory: 10.00 MB\n\nOK (1 test, 1 assertion)","coveredBy":["97ea2e5ee1e0a71c3f2a240c57ba2162"],"killedBy":[],"testsCompleted":0},{"id":"d136f1faa27eac8c072f6e8070be3e14","mutatorName":"IncrementInteger","replacement":"return $mixture + 3;","description":"Increments an integer value with 1.","location":{"start":{"line":12,"column":9},"end":{"line":12,"column":29}},"status":"Killed","statusReason":"PHPUnit 10.5.1 by Sebastian Bergmann and contributors.\n\nRuntime:       PHP 8.1.12-1ubuntu4.3\nConfiguration: \/tmp\/infection\/phpunitConfiguration.d136f1faa27eac8c072f6e8070be3e14.infection.xml\n\n.                                                                   1 \/ 1 (100%)\n\nTime: 00:00.002, Memory: 10.00 MB\n\nOK (1 test, 1 assertion)","coveredBy":["97ea2e5ee1e0a71c3f2a240c57ba2162"],"killedBy":[],"testsCompleted":0},{"id":"ddf10fadb3cfffee67afdf0813d4a1a7","mutatorName":"Plus","replacement":"return $mixture - 2;","description":"Replaces an addition operator (`+`) with a subtraction operator (`-`).","location":{"start":{"line":12,"column":9},"end":{"line":12,"column":29}},"status":"Killed","statusReason":"PHPUnit 10.5.1 by Sebastian Bergmann and contributors.\n\nRuntime:       PHP 8.1.12-1ubuntu4.3\nConfiguration: \/tmp\/infection\/phpunitConfiguration.ddf10fadb3cfffee67afdf0813d4a1a7.infection.xml\n\n.                                                                   1 \/ 1 (100%)\n\nTime: 00:00.002, Memory: 10.00 MB\n\nOK (1 test, 1 assertion)","coveredBy":["97ea2e5ee1e0a71c3f2a240c57ba2162"],"killedBy":[],"testsCompleted":0}]}},"testFiles":{"\/home\/barney\/projects\/infection-static-analysis-plugin\/test\/asset\/ProjectWithPsalmBaseline\/tests\/HelloWorldTest.php":{"tests":[{"id":"97ea2e5ee1e0a71c3f2a240c57ba2162","name":"HelloWorldTest::testNothingButProvideCoverage"}]}},"framework":{"name":"Infection","branding":{"homepageUrl":"https:\/\/infection.github.io\/","imageUrl":"https:\/\/infection.github.io\/images\/logo.png"}}}
```

Comment:
Do we really need to commit this stuff? :thinking:

Reply:
apologies, this one was committed by mistake.

Reply:
Removed in [5e66e68](https://github.com/Roave/infection-static-analysis-plugin/pull/489/commits/5e66e68b0f8d9da3aa89565e0a03a9a5907ef574)